src/dune_rules/jsoo_rules.ml

```diff
@@ -72,6 +72,53 @@ end = struct
       | name, false -> [ "--disable"; name ])
 end
 
+module Version = struct
+  type t = int list
```

Comment:
Shouldn't this be `int * int * int`? And in particular, only the first 2 number should matter for detection.

Reply:
I've changed it to `int * int`